certificate-service/app/src/main/java/se/inera/intyg/certificateservice/application/unit/dto/UnitStatisticsRequest.java

```diff
@@ -0,0 +1,27 @@
+package se.inera.intyg.certificateservice.application.unit.dto;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import java.util.List;
+import lombok.Builder;
+import lombok.Value;
+import se.inera.intyg.certificateservice.application.common.dto.UnitDTO;
+import se.inera.intyg.certificateservice.application.common.dto.UserDTO;
+import se.inera.intyg.certificateservice.application.unit.dto.UnitStatisticsRequest.UnitStatisticsRequestBuilder;
+
+@JsonDeserialize(builder = UnitStatisticsRequestBuilder.class)
+@Value
+@Builder
+public class UnitStatisticsRequest {
+
+  UserDTO user;
+  UnitDTO unit;
+  UnitDTO careUnit;
+  UnitDTO careProvider;
+  List<String> unitIds;
```

Comment:
Since it's a list of all available units for the user, I updated it to availableUnitIds, I hope that makes and clearer communicate the content of the list.